### DISCUSSION TITLE
add a public switch, which can be viewed without logging in

### DISCUSSION THREAD

**Discussion by fengdu126** [docmost/docmost#282] [Ideas]
![image](https://github.com/user-attachments/assets/5cb3c3bf-9e4a-473d-b9e6-e8aa7215c7d6)

**Comment by fengdu126** [comment]
It would be nice to be able to choose which space is publicly accessible

**Comment by almereyda** [comment]
Good idea. Where did you take this screenshot from?

**Comment by Dave-theGreat** [comment]
I would love to have this feature.

**Comment by darktweek** [comment]
+1 for the idea

**Comment by darktweek** [comment]
Any plan to add this feature ?

**Comment by excxapp** [comment]
+1 for the idea
